src/providers/utils.ts

```diff
@@ -0,0 +1,91 @@
+// The async/queue library has a task-based interface for building a concurrent queue.
+
+import { providers } from "ethers";
+import lodash from "lodash";
+import { compareArrayResultsWithIgnoredKeys, compareResultsAndFilterIgnoredKeys } from "../utils/ObjectUtils";
+
+export { lodash };
```

Comment:
I think we only use relatively few things from lodash here - is it worth only exporting the few things that we need?

Reply:
https://github.com/across-protocol/sdk/pull/693/commits/75528faa95972ff871ff0f36f28fafe7c814165e